chainList.json

```diff
@@ -128,8 +128,8 @@
     }
   },
   {
-    "name": "OP-Sepolia",
-    "identifier": "sepolia/op",
+    "name": "OP Sepolia Testnet",
+    "identifier": "sepolia/opsep",
```

Comment:
This name seems redundant since you have the `sepolia` prefix and the `sep` suffix

Reply:
The name is redundant in this case, but not in all cases. I actually rolled back the changes to this file in the end.

Reply:
We will likely run into this in future again, but it may be something we can't really get around.

Reply:
There are two options here:
1. Lean into the _existing_ behaviour: have folks submit i) chain name ii) chain shortname and iii) "unqualified short identifier". The config yml filename uses the unqualified short identifier, e.g. "op" for both OP mainnet and OP sepolia. We inform contributors that this USI can be the same for testnets and mainnets. We will then avoid redundant identifiers.
2. Try to remove the filename as being semantically important and an extra degree of freedom. To avoid breaking changes, we only do this for new chains: set the filename to the chain short name. The identifier will then be built from the chain short name, and we are more likely to get redundant information in the fully qualified identifier above.

Reply:
There's also option:

0: remove `shortChainName` from this PR altogether. Then this PR has minimal changes and we can implement the other options at a later time if we want to.

Reply:
Yes I'd vote option 0

Reply:
Implemented here [ea3c330](https://github.com/ethereum-optimism/superchain-registry/pull/337/commits/ea3c33047740b47b6dbdd1dd17fee838d04d0625)

I still used the "chain short name" terminology, since "unqualified short identifier" seemed a bit... much. But I made it clear in the .env.example file that it is not to be confused with the upstream chain short name.